docs/user-guide/team-planner/README.md

```diff
@@ -0,0 +1,101 @@
+4. There is a filter button by which you can focus what is displayed on your calendar (for example, only certain team members or work packages).
+5. This switches you team planning calendar to full screen mode.
+6. Saving, renaming and saving a copy (saving as), or deleting can also be accomplished using the **[⋮]** button. This is also where you can modify the visibility options. Select **public** to allow others in your organization that are members of your respective project to see your planning calendar. If **public** is not selected, the calendar is **private** and can only be seen by you, not other members.
+7. By default the planning calendar only shows the current week. Use these buttons to toggle between the 1-week and 2-week view.
+
+## Getting started
+
+- ##### Step 1: Adding team members
+
+The first step in setting up your team planning calendar is to add team members. This is done via the **+ add assignee** button in #3 above. Press the **+ add assignee** button, and then simply select the team member(s) from the drop-down list. Clicking on a given assignee's name will open a new window with a quick overview of which projects that member is assigned to.
+
+Repeat this step until all team members relevant for this planner are added, and then save it using the floppy disk icon (adjacent to the top header in #1 above). 
+
+**Note: Once you are finished adding members to the planner, click the floppy disk icon in the top header to save.**
+
+![Adding team members](adding-team-members.png)
+
+
+
+- **Step 2: Adding/working with work package strips**
+
+Next, add specific work packages to your planner. Do this by pressing the **+ add existing" button** we introduced in step 2 of the main view overview. Once you press the **+ add existing** button, start typing the name of the work package you are looking to add. Once found, drag & drop the work package strip. Should the work package have a duration exceeding the current 1 or 2 week view, a horizontal scroll bar will display, with which it is possible to scroll to the end date of the longest running work package.
+
+Additional features: You can change the status of a work package, and enter the work package strip detailed view by pressing the info button **(i)** on each respective strip
+
+**Note: You can only add existing work packages here. If you need to create some first, create them in the work package module as new work packages cannot be created from the team planner module.**
+
+![Working with work package strips](working-with-work-package-strips.png)
+
+- **Work package strip detail view**
+
+Click on any given work package strip to open up a split screen showing the work package detail view.
+
+![Work package strip detail view](work-package-strip-detail-view.png)
+
+
+
+- **Removing a work package strip**
```

Comment:
Please use headlines format, manually boldend text can't be linked to. e.g. for Level-2 headline
 `## Removing a work package strip` the anchor-id `#removing-a-work-package-strip` is auto generated, so we can link to this section.